cardano-testnet/src/Testnet/Start/Cardano.hs

```diff
@@ -100,7 +100,7 @@ cardanoTestnetDefault testnetOptions shelleyOptions conf = do
   AnyShelleyBasedEra sbe <- pure cardanoNodeEra
   alonzoGenesis <- getDefaultAlonzoGenesis sbe
   shelleyGenesis <- getDefaultShelleyGenesis cardanoNodeEra cardanoMaxSupply shelleyOptions
-  cardanoTestnet testnetOptions conf shelleyGenesis alonzoGenesis Defaults.defaultConwayGenesis
+  cardanoTestnet testnetOptions conf Nothing shelleyGenesis alonzoGenesis Defaults.defaultConwayGenesis
```

Comment:
Having a sum type here in stead of `Maybe` would make this more readable. E.g `UserConfigurationNotSubmitted`